v-next.sql

```diff
@@ -0,0 +1,493 @@
+------------------------------------------------------------
+-- add edition reference to test standards
+------------------------------------------------------------
+
+-- fix 2014 test standards
+
+--fix 170.205(a)(1): it had a duplicate and needs to be resolved to be a single entry
+-- update cert results to use one standard
+UPDATE openchpl.certification_result_test_standard 
+SET test_standard_id = 
+	(SELECT MIN(test_standard_id) FROM openchpl.test_standard WHERE number = '170.205(a)(1)' and test_standard_id <= 47)
+WHERE test_standard_id IN (SELECT test_standard_id FROM openchpl.test_standard WHERE number = '170.205(a)(1)' and test_standard_id <= 47);
+--delete any other test standards with this number
+UPDATE openchpl.test_standard 
+SET deleted = true, last_modified_user = -1
+WHERE number = '170.205(a)(1)' 
+AND test_standard_id > (SELECT MIN(test_standard_id) FROM openchpl.test_standard WHERE number = '170.205(a)(1)' and test_standard_id <= 47) 
+AND test_standard_id <= 47;
```

Comment:
~~Why the `<= 47` part? If that's the first ID for 170.205(a)(1), then why the same number for the following changes? (170.205(a)(2) et al)~~
Nevermind; that's the breakpoint before 2015 standards